pypika/tests/test_create.py

```diff
@@ -165,3 +165,42 @@ def test_create_table_with_select_and_columns_fails(self):
     def test_create_table_as_select_not_query_raises_error(self):
         with self.assertRaises(TypeError):
             Query.create_table(self.new_table).as_select("abc")
+
+
+class CreateIndexTests(unittest.TestCase):
+    new_index = Index("abc")
+    new_table = Table("my_table")
+    columns = (new_table.a, new_table.b)
+    where = new_table.a > 1
+
+    def test_create_index_simple(self) -> None:
+        q = Query.create_index(self.new_index).on(self.new_table).columns(*self.columns)
+        self.assertEqual(
+            f'CREATE INDEX "{self.new_index.name}" ON "{self.new_table.get_table_name()}"({", ".join([c.name for c in self.columns])})',
+            str(q.get_sql()))
```

Comment:
is `str` conversion needed if `get_sql` method already returns a `str`?

Reply:
I know but when I wrote the tests I did not have the __str__ and __repr__ overloads. will remove those

Reply:
Actually, this has nothing to do with overloads. Rather copied from somewhere and forgot to remove the `str()`